libs/jsonschema/extension.go

```diff
@@ -26,4 +26,8 @@ type Extension struct {
 	// If the CLI version is less than this value, then validation for this
 	// schema will fail.
 	MinDatabricksCliVersion string `json:"min_databricks_cli_version,omitempty"`
+
+	// Skip prompting if this schema is satisfied by the user entered values. In
```

Comment:
Nit: the values to check don't need to be user entered.

```suggestion
	// Skip prompting if this schema is satisfied by the configuration already present. In
```

Reply:
done